src/components/settings/SettingsAccessTab/DisableLinkManagementAlertModal.jsx

```diff
@@ -1,45 +1,22 @@
-import React, { useState } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import {
   ActionRow,
   AlertModal,
+  Alert,
   Button,
   StatefulButton,
 } from '@edx/paragon';
-import { logError } from '@edx/frontend-platform/logging';
+import { Info } from '@edx/paragon/icons';
 
 const DisableLinkManagementAlertModal = ({
   isOpen,
   onClose,
-  onDisableLinkManagement,
+  onDisable,
+  isLoadingDisable,
```

Comment:
nit: do you think we can get away with just isLoading here? isLoadingDisable is not intuitive and looks like it could be typo.